test/rlcIntegration/generated/lroRest/tsconfig.json

```diff
@@ -19,8 +19,7 @@
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
     "outDir": "./dist-esm",
-    "declarationDir": "./types",
-    "paths": { "@msinternal/lro-rest": ["./src/index"] }
+    "declarationDir": "./types"
   },
-  "include": ["./src/**/*.ts", "samples-dev/**/*.ts"]
```

Comment:
why do we have this if we are removing sampleConfiguration in the package.json

Reply:
This is the case where we enable the sampleGenerated option but no examples provide in testserver. I'll add examples in phase 2 for this swagger.